ddtrace/contrib/dbapi/__init__.py

```diff
@@ -34,7 +35,16 @@ def __init__(self, cursor, pin, cfg):
         name = pin.app or 'sql'
         self._self_datadog_name = '{}.query'.format(name)
         self._self_last_execute_operation = None
-        self._self_config = cfg or config.dbapi2
+        if not cfg:
```

Comment:
🤔 when is this condition actually met? I think all our dbapi integrations pass their own configs. Just concerned that this will be extremely noisy or not at all effective.

Reply:
This is only in case someone out there was using dbapi2 integration directly.

Reply:
Gotcha - yeah that makes sense